### ISSUE TITLE
Inflation Adjustment?

### ISSUE THREAD

**Issue by kamilkrukowski** [kamilkrukowski/sec-sentiment-pred#11]
The USA publishes monthly inflation figures for the past ~25 years.

S&P500 annualized returns are about ~11% without inflation and about ~7% with inflation.

Current Prime Rate is 7.75% (The most credit-worthy institutional lending rate). Since we look at a 2023 prime rate, we need to take into account the inflation eating away at our returns prior to 2023. (This is noticeable because the pandemic has doubled inflation during the course of COVID and that increase is maintained today, up at 6.41% today).

So in short, if we have an ROI of 8% percent today, we are actually only expecting a (~1.6%) inflation-adjust return. This is incredibly important to point out and analyze if we want to actually project the results.

**Comment by yyitingli** [comment]
Consumer Price Index for All Urban Consumers: All Items in U.S. City Average can be downloaded [https://tufts.box.com/s/565js9mmll075a0uzylj83ljgm5gb7m1](url)

**Comment by kamilkrukowski** [comment]
Looks good!
